Server/DB_Connection/adminQueries.py

```diff
@@ -0,0 +1,131 @@
+import json
+from DB_Connection.dbConnect import DBConnection
+from decimal import Decimal
+import traceback
+
+class AdminQuery:

```

Comment:
can you think of a better name for DB layer classes? you can refer to the articles shared during the workshop